### PR TITLE
New/fixed autoconfigs: Requested changes from pull request #719

### PR DESCRIPTION
**Pull request by mrmatteastwood**
This is a pull request for the changes I made to the files I submitted in pull request #719.

Please bear in mind that in the udev folder, the following file needs to be deleted:
Afterglow_Gamepad_for_Xbox_360.cfg

It contains the incorrect mapping for some buttons. When that file is present, RetroArch loads it instead of the correct one I created (using RetroArch).

See more info regarding that in my comments in pull request #719.

### CODE REVIEW

File: sdl2/Afterglow.cfg [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# [2021.01.24] Manufacturer's product page for the Afterglow Xbox 360 wired pad no longer exists
+# Product page at Walmart: https://www.walmart.com/ip/PDP-Afterglow-Wired-Controller-Blue-for-Xbox-360/33130220
+
 input_driver = "sdl2"
 input_device = "Afterglow"
```
Comment (RobLoach) [inline sdl2/Afterglow.cfg:5]:
``` suggestion
input_device = "Afterglow"
input_device_display_name = "PDP Afterglow Wired Controller"
```

File: sdl2/Generic X-Box pad.cfg [hunk 2]
```diff
@@ -1,3 +1,5 @@
+# Some PDP Afterglow wired Xbox controllers (https://www.walmart.com/ip/PDP-Afterglow-Wired-Controller-Blue-for-Xbox-360/33130220) might be recognized as "Generix X-Box pad" by RetroArch using the sdl2 driver. This is the autoconfig for that.
+
 input_driver = "sdl2"
 input_device = "Generic Xbox controller"
```
Comment (RobLoach) [inline sdl2/Generic X-Box pad.cfg:4]:
``` suggestion
input_device = "Generic Xbox controller"
input_device_display_name = "Generic Xbox Controller"
```

File: sdl2/Logitech F310 Gamepad (XInput).cfg [hunk 3]
```diff
@@ -1,3 +1,5 @@
+# Manufacturer's product page: https://www.logitechg.com/en-us/products/gamepads/f310-gamepad.940-000110.html
+
 input_driver = "sdl2"
 input_device = "Logitech F310 Gamepad (XInput)"
```
Comment (RobLoach) [inline sdl2/Logitech F310 Gamepad (XInput).cfg:4]:
``` suggestion
input_device = "Logitech F310 Gamepad (XInput)"
input_device_display_name = "Logitech F310 Gamepad"
```

File: udev/Afterglow Gamepad for Xbox 360.cfg [hunk 4]
```diff
@@ -1,3 +1,6 @@
+# [2021.01.24] Manufacturer's product page for the Afterglow Xbox 360 wired pad no longer exists
+# Product page at Walmart: https://www.walmart.com/ip/PDP-Afterglow-Wired-Controller-Blue-for-Xbox-360/33130220
+
 input_driver = "udev"
 input_device = "Afterglow Gamepad for Xbox 360"
```
Comment (RobLoach) [inline udev/Afterglow Gamepad for Xbox 360.cfg:5]:
``` suggestion
input_device = "Afterglow Gamepad for Xbox 360"
input_device_display_name = "PDP Afterglow Wired Controller"
```

**Review by RobLoach — APPROVED** [review]
This is most excellent, thank you. What are your thoughts on the display names for each? Are these kind of right?

**Comment by mrmatteastwood** [comment]
Hey Rob, yeah, sounds good! For the Afterglow, perhaps add "Xbox 360": "PDP Afterglow Wired Xbox 360 Controller". I'll remember the input_device_display_name option for if I do more autoconfigs in the furue :-)

**Pull request merged**